### PR TITLE
docs(onboarding): add coding-agent PR etiquette guide

### LINKED ISSUE #1770: docs(onboarding): add coding-agent PR etiquette guide
## Description
Implement an LLM-friendly, independently shippable backlog item for **add coding-agent PR etiquette guide**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- docs/onboarding and contributor guides
- CLI/help output or onboarding scripts
- PM/worker handoff templates

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add coding-agent PR etiquette guide as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add coding-agent PR etiquette guide is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add coding-agent PR etiquette guide.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Onboarding**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- Add a dedicated coding-agent PR etiquette guide under `docs/onboarding/` covering one-issue/one-PR scope, PR body evidence, review/Codex freshness, and blocked handoff fields.
- Link the guide from `ONBOARDING.md` and the README onboarding entrypoint while preserving the current PM-swarm runtime glossary links from main.
- Add deterministic docs tests for the guide, onboarding/README links, and negative PR etiquette edge cases.

## Verification
- `npx vitest run tests/docs-issue-1770.test.ts tests/docs-issue-1768.test.ts tests/docs-issue-1094.test.ts` — 10 tests passed
- `npm run typecheck` — 17 tasks successful
- `npm run build` — 10 tasks successful

## Scope and handoff
- Issue: Closes #1770
- Branch: `resolve/issue-1770-docs-onboarding-add-coding-agent-pr-etiquette-gu`
- Ownership entries: `onboarding-docs`
- Codex: pending current-head `@codex review`

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/coding-agent-pr-etiquette.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```text
+Current head: <sha>
+CI: green | failing <check-name> | pending
+Codex: clean | findings unresolved | usage-limited | not triggered
+Blocked command, if any: <exact command>
+Next safe command: <command that continues the same PR, not a duplicate branch>
+Verification already run: <commands and results>
+```
+
+For ownership routing, read the repository ownership manifest and include matching entry ids when a PR spans packages, docs, scripts, or workflows. For docs-only onboarding changes, `docs-onboarding` is the expected ownership surface.
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/coding-agent-pr-etiquette.md]: Fixed in c698730d by changing the docs-only onboarding ownership surface from the non-existent `docs-onboarding` id to the manifest-backed `onboarding-docs` id, and added a deterministic test assertion so the guide continues to name the real manifest id.

**Comment by djm204** [comment]
@codex review

**Comment by djm204** [comment]
@codex review
